frontend/packages/network-attachment-definition-plugin/src/types/types.ts

```diff
@@ -21,6 +21,8 @@ export type NetworkAttachmentDefinitionConfig = {
   type?: string;
   bridge?: string;
   isGateway?: true;
+  macspoofchk?: boolean;
+  vlan?: number;
```

Comment:
where do use vlan ? did we introduce it in this PR ?

Reply:
No it's part of cnv bridge. Because we don't use object assign it's need declaration now

Reply:
It's vlan tag number. I think it's 2 years there..

Reply:
nice +1